src/components/utils/popover.ts

```diff
@@ -7,9 +7,9 @@ import { isMobileScreen, keyCodes, cacheable } from '../utils';
 import ScrollLocker from './scroll-locker';
 
 /**
- * Describe parameters for rendering the single item of Popover
+ * Describe parameters for rendering a single popover item
  */
-export interface PopoverItem {
+export interface PopoverItemBase {
```

Comment:
Do we really need to export this interface?

Reply:
removed export